### PR TITLE
If given location is latitude/longitude, do not geolocate

### PR DESCRIPTION
**Pull request by andythorne**
### Short Description:

When you provide a latitude and longitude location (`xx.xxxxxxx,yy.yyyyyyy`), the geolocation snaps the location to the nearest address. If you want to start in the middle of a park/forest, it causes unwanted behaviour.
### Changes:
- If coordinates are provided, only call google's elevation API for the altitude. If a named location is given, use geolocation.

@OpenPoGo/maintainers

### CODE REVIEW

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -423,6 +424,28 @@ def _read_config_location(self):
...
+                response = gmaps.elevation((pos_lat, pos_lng))
+
+                if len(response) and "elevation" in response[0]:
+                    return pos_lat, pos_lng, response[0]["elevation"]
+                else:
+                    raise ValueError
+            except ApiError:
+                logger.log("[x] Could not fetch altitude from google. Trying geolocator.")
+            except ValueError:
+                logger.log("[x] Location was not Lng/Lat.")
```
Comment (MaxLeiter) [inline pokemongo_bot/__init__.py]: bit of a nitpick, but mind switching to "[x] Location was not **Lat/Lng**"? (bold used as diff) 

Same with comment below

Comment (andythorne) [inline pokemongo_bot/__init__.py]: Good spot! Updated :)

**Comment by MaxLeiter** [comment]
:+1:  Thanks!

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/114/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by jtdroste** [comment]
👍 Gonna let another person merge

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/114/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**